src/main/java/managers/DriverManager.java

```diff
@@ -74,6 +67,18 @@ public WebDriver getDriver() {
         return driver;
     }
 
+    public void waitForPageToLoad() {
+        ExpectedCondition<Boolean> isLoaded = (WebDriver driver) ->
+                ((JavascriptExecutor) driver).executeScript("return document.readyState").toString().equals("complete");
+
+        try {
+            Thread.sleep(1000);
```

Comment:
you can use eg fluent wait, not thread sleep. And what if during tests page will be reloaded? and why 1000ms?